modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -252,9 +253,10 @@ private void _validate(
 			}
 		}
 
-		if (Objects.equals(
-				objectActionTriggerKey,
-				ObjectActionTriggerConstants.KEY_ON_AFTER_DELETE) &&
+		if ((!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153714")) ||
```

Comment:
There is no need to include the FF in this condition because there is no bad impact in this validation.
I think we need to add the FF validation in the POST method in the ObjectActionResourceImpl:
`if ((!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153714")) &&
	Objects.equals(objectActionExecutorKey,ObjectActionExecutorConstants.KEY_UPDATE_OBJECT_ENTRY)) {
		throw new UnsupportedOperationException();
}`

this exception can be generic.